book-1-orientation/chapters/QUEST.md

```diff
@@ -0,0 +1,287 @@
+        public class Adventurer
+        {
+            // The is an "immutable" property. It only has a "get".
+            // The only place the Name can be set is in the Adventurer constructor
+            public string Name { get; }
+
+            // This is a mutable property it has a "get" and a "set"
+            //  So it can be read and changed by any code in the application
+            public int Awesomness { get; set; }
+
+            // A constructor to make a new Adventurer object with a given name
+            public Adventurer(string name)
+            {
+                Name = name;
+                Awesomness = 50;
+            }
+
+
+            // This method returns a string that describes the Adventurer's status
+            // Note one way to describe what this method does is:
+            //   it transforms the Awesomeness integer into a status string
+            public string GetAdventurerStatus()
+            {
+                string status = "okay";
+                if (Awesomness >= 75)
+                {
+                    status = "great";
+                }
+                else if (Awesomness < 25 && Awesomness >= 10)
+                {
+                    status = "not so good";
+                }
+                else if (Awesomness < 10 && Awesomness > 0)
+                {
+                    status = "barely hanging on";
+                }
+                else if (Awesomness <= 0)
+                {
+                    status = "not gonna make it";
+                }
+
+                return $"Adventurer, {Name}, is {status}";
+            }
+        }
+    }
+    ```
+
+    > `Challenge.cs`
+
+    ```cs
+    using System;
+
+    namespace Quest
+    {
+        // An instance of the Challenge class is occurrence of a single challenge
+        public class Challenge
+        {
+            // These private fields hold the "state" of an individual challenge object.
+            // The values stored in these fields are not accessible outside the class,
+            //  but can be used by methods or properties within the class
+            private string _text;
+            private int _correctAnswer;
+            private int _awesomenessChange;
+
+
+            // A constructor for the Challenge
+            // We can tell it's a constructor because it has the same name as the class 
+            //   and it doesn't specify a return type
+            // Note the constructor parameters and what is done with them
+            public Challenge(string text, int correctAnswer, int awesomenessChange)
+            {
+                _text = text;
+                _correctAnswer = correctAnswer;
+                _awesomenessChange = awesomenessChange;
+            }
+
+            // This method will take a Adventurer object and make that Adventurer perform the challenge
+            public void RunChallenge(Adventurer Adventurer)
```

Comment:
adventurer rather than Adventurer for the parameter?
Will require changes to lines 201, 206, and 210 if changed.